build_tools/cmake/iree_copts.cmake

```diff
@@ -417,6 +417,7 @@ iree_select_compiler_opts(IREE_DEFAULT_LINKOPTS
     ${_IREE_LOGGING_LINKOPTS}
   MSVC
     "-natvis:${IREE_ROOT_DIR}/runtime/iree.natvis"
+    "-pdbpagesize:32768"
```

Comment:
Probably worth a comment here. Could link to https://github.com/iree-org/iree/issues/20763, or explain a bit about why this flag is added.

Documenting each flag makes it easier to know when that flag is safe to remove, or why another project might want to include it.